app/components/spotlight/analytics/aggregation_component.html.erb

```diff
@@ -0,0 +1,14 @@
+<table class="table analytics">
+  <tr>
+  <% display_fields.each_pair do |key, value| %>
```

Comment:
When there's no data available from GA for an exhibit the app errors here with `undefined method `each_pair' for []:Array`. I think we need to handle the case where there's no data yet for a particular exhibit. I assume the same thing would happen on line 9.

Reply:
The render? statement I put in here should have fixed that.